content/en/security/threats/workload_security_rules/custom_rules.md

```diff
@@ -16,13 +16,64 @@ further_reading:
+
+There are two use cases:
+
+- **Create a detection rule using an existing Agent rule:** To create a threat detection rule that uses an existing Agent rule, you only need to create a threat detection rule that references the Agent rule and adds any additional expression parameters you need.
+- **Create a threat detection rule using a new Agent rule:** To detect an event that the current Agent rules do not support, you need to create a custom Agent rule to detect that event, and then create a custom threat detection rule that uses the custom Agent rule.
+
+For more information, see [CSM Threats Detection Rules][7]. 
+
+You can create custom rules using these methods:
+
+- **Simple:** Use the **Simple rule creator** to create the custom Agent and detection rules together.
+  - For steps on using the **Simple rule creator**, see [Create the custom agent and detection rules together](#create-the-custom-agent-and-detection-rules-together).
+- **Advanced:**  Create custom Agent and detection rules individually by defining their threat detection expressions. 
+  - For steps on this method, see [Create a custom agent rule](#create-a-custom-agent-rule) and [Create a custom detection rule](#create-a-custom-detection-rule).
+
+## Create the custom agent and detection rules together
+
+The **Simple rule creator** option helps you create the Agent and dependent detection rules together, and ensures that the Agent rule is referenced in the detection rules. Using this tool is faster than the advanced method of creating the Agent and detection rules separately.
+
+As you define the rules using this tool, the threat expressions generated for these rules are displayed in the tool.
+
+To use the Simple rule creator:
+
+1. In [Agent Configuration][4] or [Threat Detection Rules][3], select **New Rule**, and then select **Simple rule creator**.
+2. Define the detection.
+   - To detect unauthorized changes to files, select **File integrity monitoring (FIM)**.
+   - To track and analyze system software processes for malicious behavior or policy violations, select **Process activity monitoring**.
+   - Enter the file/process names or paths to monitor. 
+3. Specify more conditions. Enter any arguments to add to the threat rule expression. For example, the argument `foo` is added as `process.argv in ["foo"]`.
+4. Set severity and notification lists. 
+   - Select the severity for the signal generated when this threat is detected. 
+   - Select notification lists to notify when a signa is generated.
+5. Add rule name and description.
+   
+   Here's an example of a new FIM rule, including the expressions generated for each rule.
+
+    {{< img src="/security/csm/csm_threats_simple_rule_creator.png" alt="Simple rule creator example" style="width:100%;" >}}
+
+6. Select **Create `<<NUMBER>>` Rules**.
```

Comment:
```suggestion
6. Select **Create `<NUMBER>` Rules**.
```
This came up in backroom today, but a simpler convention would be **Create _N_ Rules**, if you prefer that.